tests/basic/volume_ops_test.go

```diff
@@ -2843,6 +2843,111 @@ var _ = Describe("{NFSProxyVolumeValidation}", func() {
 	})
 })
 
+var _ = Describe("{SharedVolFuseTest}", func() {
+	/*
+
+				https://portworx.atlassian.net/browse/PWX-35639
+			   https://portworx.atlassian.net/browse/PTX-21805
```

Comment:
Can we add testrail id in the code
	var testrailID = 12133434
	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/35259
	var runID int
	JustBeforeEach(func() {
		StartTorpedoTest("VolumeDriverDown", "Validate volume driver down", nil, testrailID)
		runID = testrailuttils.AddRunsToMilestone(testrailID)
	})
	var contexts []*scheduler.Context

Reply:
fixed